src/ShellCheck/Interface.hs

```diff
@@ -21,7 +21,7 @@
 module ShellCheck.Interface
     (
     SystemInterface(..)
-    , CheckSpec(csFilename, csScript, csCheckSourced, csExcludedWarnings, csShellTypeOverride, csMinSeverity)
+    , CheckSpec(..)
```

Comment:
The fields were listed out to force anyone importing the library to use `newCheckSpec` rather than `CheckSpec` to construct a new spec. That way, any changes like this will be API compatible with older versions

Reply:
I wondered if there was a reason for this! I'll change it back and add the new field to the exports 👍